src/client/conn.rs

```diff
@@ -462,6 +464,24 @@ impl Builder {
         self.h1_read_buf_exact_size = sz;
         self
     }
+
+    /// Set the maximum buffer size for the connection.
+    ///
+    /// Default is ~400kb.
```

Comment:
We should probably document that setting this option conflicts with setting the `http1_read_buf_exact_size` option.